### PR TITLE
site policy.

### PR DESCRIPTION
**Pull request by ctb**

### CODE REVIEW

File: src/2015-site-policies.rst [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+I'm open to suggestions for additions and removals; please post them
+in comments.  However, this set of policies stands until I change them -
+thanks for abiding by them!
+
+Copyright Notice
+----------------
+
+I own the copyright to all of my writing. Guest bloggers retain their
+own copyright. Quote all of us freely!  Link & attribute always!
```
Comment (gvwilson) [inline src/2015-site-policies.rst]: Given "link & attribute always", should you explicitly mention CC-BY here? (Is that the license?)

File: src/2015-site-policies.rst [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+Questions
+---------
+
+By posting your comments to the blog you are granting me the right, in
+perpetuity, to use, quote, alter, and/or display those questions and
```
Comment (gvwilson) [inline src/2015-site-policies.rst]: "alter" ??  Um...

File: src/2015-site-policies.rst [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+I do not share your personal email address or identity, with one
+exception: People who threaten or harass me or other posters will have
+their names, emails, and IP addresses shared with their internet
+service providers and, if necessary, law enforcement.
+
+Comments Policy
+---------------
+
+We moderate things in these here parts. Not every point of view
+deserves to be heard. Unmoderated spaces tend to be full of assholes.
```
Comment (gvwilson) [inline src/2015-site-policies.rst]: s/assholes/something-you-would-not-consider-impolite-if-directed-at-you-in-a-comment-on-your-blog/

**Comment by mr-c** [comment]
Seems sufficiently effective and Titus-y.

While I'm super-OK with aggro directed at bad actors, this policy may scare off those who haven't weathered internet flame wars. Perhaps a sentence or two about how you do value the vast majority of comments the blog; how they fill you with joy and butterflies, et cetera.

File: src/2015-site-policies.rst [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+- Sarcasm, curse words, gentle mocking, criticism and vehement
+  disagreement are generally allowed, as long as the overall spirit is
+  constructive and in good faith.
+
+- You're otherwise being useless, uninformative, or destructive.
+
+Having your comment deleted or being asked to stop commenting in a
+particular thread or to move on from a topic of discussion is IN NO
+WAY an invitation for you to send me lengthy emails detailing your
+thoughts. If I wanted to read Your Thoughts I would have let you keep
```
Comment (gvwilson) [inline src/2015-site-policies.rst]: s/Your Thoughts/your thoughts/

**Comment by ctb** [comment]
+1

File: src/2015-site-policies.rst (hunk 2 above)
Comment (ctb) [inline src/2015-site-policies.rst]: yep. I mean, I won't, but.

File: src/2015-site-policies.rst (hunk 1 above)
Comment (ctb) [inline src/2015-site-policies.rst]: CC0; I will add.

File: src/2015-site-policies.rst (hunk 2 above)
Comment (mr-c) [inline src/2015-site-policies.rst]: Possible reference to the "fart fart fart" comment moderation style? http://sarah.thesharps.us/2015/10/05/closing-a-door/#comment-147828

Comment (ctb) [inline src/2015-site-policies.rst]: thx, added.

**Pull request merged**